libs/constants/tokenList.ts

```diff
@@ -11,6 +11,12 @@ const tokenList: Record<AvailableNetwork, StaticTokenInfo[]> = {
             decimals: 6,
             address: '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8',
         },
+        {
+            name: 'FRAX',
+            symbol: 'FRAX',
+            decimals: 18,
+            address: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
```

Comment:
pretty sure FRAX on arbitrum is `0x17fc002b466eec40dae837fc4be5c67993ddbd6f` I'll check I haven't got this in the SDK too